src/system/documents/item.ts

```diff
@@ -555,12 +583,12 @@ export class CosmereItem<
                 user: game.user!.id,
                 speaker,
                 content: `<p>${flavor}</p>`,
-                rolls: [skillRoll, damageRoll],
+                rolls: [skillRoll, damageRolls],
```

Comment:
Similar to my other comment I think it's better here to just collate all rolls into a flat array of rolls because it'll be way easier to manipulate when creating the chat message styling. Some chat message properties are also already set up with a flat array in mind (e.g. the damage rolls property on a chat message searches the array for all objects instanceof DamageRoll, and is not expecting DamageRol[])

Reply:
Ah, I meant to add the spread operator in here.
Will do a quick push

Reply:
done, silly thing was the damage-only branch was already doing this 🤦🏻‍♂️